src/views/Downloads.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div class="container">
+      <v-card :key='download.title'
+        v-for='download in downloads' class="card">
+        <a :href="`${publicPath}downloads/${download.fileName}`" download class="cardContainer">
+          <div>
+            <v-icon class="material-icons icon" color='black' size='15vh'>
+              system_update_alt
+            </v-icon>
+            <v-divider class='divider'/>
+            {{download.title}}
+          </div>
+        </a>
+      </v-card>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+import Download from '@/models/Download';
+import { AllDownloads } from '@/db-service/Downloads/queries';
+
+@Component({
+  apollo: {
+    allDownloads: {
+      query: AllDownloads,
+    },
+  },
+  data() {
+    return {
+      publicPath: process.env.BASE_URL,
```

Comment:
You can put the data inside the class, the target of putting it in is making the component OOP...